src/user/App.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Route, Switch } from "react-router-dom";
+import Profile from "./views//Profile";
+import { NavBar } from "src/partner/modules/NavBar";
+
+interface IUserAppProps {
+  match: {
```

Comment:
create an interface `IReactRouterProps` and add it under `common`

Reply:
is common going to be the shared directory?

Reply:
yep, it is gonna be